src/utils/input/index.js

```diff
@@ -57,6 +58,31 @@ const getInputsFromSources = async ({
 				})
 			: stdIn.secondPassphrase || null;
 
+	const passphraseErrors = [passphrase, secondPassphrase].reduce(
+		(accumulator, input) => {
+			if (input) {
+				const errors = elements.passphrase.validation
+					.getPassphraseValidationErrors(input)
+					.filter(error => error.message);
+				if (accumulator.length === 0) {
+					return errors;
+				}
+				return [...accumulator, ...errors];
+			}
+			return accumulator;
+		},
+		[],
+	);
+
+	if (passphraseErrors.length > 0) {
+		const uniquePassphraseErrors = [...new Set(passphraseErrors)].filter(
+			error => error.code !== 'INVALID_MNEMONIC',
+		);
+		uniquePassphraseErrors.forEach(error =>
+			console.info(`Warning: ${error.message}`),
```

Comment:
prefer not using the `console` 
Validate right before executing the command?

Reply:
process.stdout?

Reply:
I mean return back the errors from the utils and handle it with `this.print` in each command?